src/cdk-experimental/scrolling/virtual-scroll-viewport.spec.ts

```diff
@@ -489,10 +503,12 @@ describe('CdkVirtualScrollViewport', () => {
       expect(contentWrapper.children.length).toBe(4,
           'should render 4 items to fill 200px space based on 50px estimate from first item');
     }));
-
+    
     // TODO(mmalerba): Add test that it corrects the initial render if it didn't render enough,
     // once it actually does that.
-  });
+    expect((viewport._renderedContentTransform as any).changingThisBreaksApplicationSecurity)
```

Comment:
did this get copied down here by mistake?

Reply:
Yes - sorry.